app/containers/pipeline/parts/functional/InitialPreprocessing.js

```diff
@@ -75,6 +75,23 @@ class InitialPreprocessing extends Component {
               </FormControlLabelled>
             </Help>
           </FormGroup>
+
+          <FormGroup row>
+            <Help
+              type="pipeline"
+              regex={/^runDespike/}
+              help={`Turn on to run AFNI 3dDespike after motion correction`}
```

Comment:
We can go ahead and forget this toggle- we are placing 3dDespike in only one place.

Reply:
Or rather, can drop the "after motion correction" bit. We will run it before slice-timing correction and after the motion estimate calculation (if that is done before STC).